recipe_scrapers/maangchi.py

```diff
@@ -0,0 +1,59 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._utils import normalize_string
+
+
+class Maangchi(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "maangchi.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        before = self.soup.find("h4", string="Directions").find_all_previous("li")
+        after = self.soup.find("h4", string="Ingredients:").find_all_next("li")
+        lstBfr = [normalize_string(b.get_text()) for b in before]
+        lstAfr = [normalize_string(a.get_text()) for a in after]
+        lst = []
+
+        # iterate to reverse the list
+        for i in lstBfr:
+            # reversing the list
+            lst.insert(0, i)
+        lstBfr = lst
```

Comment:
(optional) a slightly easier way to do this:

```suggestion
        lstBfr.reverse()
```

Reply:
big help, thanks 👍

Reply:
could I get the `hacktoberfest-accepted` labels please, thanks

Reply:
Yep!  Labels added.